packages/create-plugin/src/utils/utils.console.ts

```diff
@@ -22,6 +22,10 @@ export function printSuccessMessage(msg: string) {
   console.log(displayAsMarkdown(`\n✔ ${msg}`));
 }
 
+export function printError(error: string) {
+  console.error(displayAsMarkdown(`\n✔ ${error}`));
```

Comment:
We should use something else here because`✔` is not a good indication of error. `❌` for example.